app.json

```diff
@@ -16,5 +16,18 @@
       "command": "python manage.py runjobs monthly",
       "schedule": "@monthly"
     }
-  ]
+  ],
+  "healthchecks": {
+    "web": [
+      {
+        "name": "Web health check",
+        "description": "Check if the app responds to the /health-check endpoint",
+        "path": "/health-check",
+        "port": 8000,
```

Comment:
thought: We didn't have to configure the port before. The [current Dokku documentation](https://dokku.com/docs/deployment/zero-downtime-deploys/#customizing-checks) says the health check defaults to `5000` in this new health check configuration. We expose port 8000 via the Dockerfile.

I'm not sure if there's some other magic that would remove the need for specifying the port.

Reply:
If it works then I think it's fine for us to specify it here